docs/dev/error-messages.md

```diff
@@ -241,6 +241,28 @@ This isn't always restricted to the tests, either; an app under test which consu
 
 Break out your long tests into shorter tests and/or make sure that your tests are not filling up a lot of disk space on the VM.
 
+
+### Unable to Find Device Within 15 Minutes
+
+**Description**
+
+This timeout occurs most frequently when you include a specific device in your test case and that device isn’t available within 15 minutes.
+
+**Cause(s)**
+
+Our public device pool is available to all subscribed Sauce users and (as you might imagine), some devices are more popular than others. We have over 280 device configurations and thousands of devices hosted in our data center, but sometimes a test queue will build up for the most popular devices.
+
+**How to Resolve**
+
+Instead of passing a specific `deviceName`:
```

Comment:
Let's give examples to make this clear. Secondly, this will happen more if customers add deviceIds instead of device names, or if their capabilities are to strict, meaning with an exact OS version and other capabilities

**Examples:**
```
// Only a deviceId
"appium:deviceName":"iPhone_12_14_real"
//
// or display name with a detailed OS version
"appium:deviceName":"iPhone 12"
"appium:platformVersion":"15.4.1"
```